package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "formatted-input",
-  "version": "0.1.1",
+  "version": "1.1.0",
```

Comment:
The version prior to my changes in NPM and in CB Components was 1.0.0, so this is intended to represent a minor version bump.